SenderEmail/src/main/java/com/example/SenderEmail/exception/ErrorResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.example.SenderEmail.exception;
+
+public class ErrorResponse {
+    private int statusCode ;
+    private  String  message ;
+
+
+    public ErrorResponse(int statusCode, String message) {
```

Comment:
- Use `final` even for parameters.
- The practice of using `final` give you more robust code and it's recommended practice by Checkstyle & Allam.